### PR TITLE
Enable CORS for pay.webpack UI development (#115)

### PR DESCRIPTION
**Pull request by kumar303**
Supports https://github.com/mozilla/payments-service/issues/115

I changed a few URLs (sign-in and subscriptions) to test it out and it seems to be working.

### CODE REVIEW

**Comment by kumar303** [comment]
TODO: 
- the UI needs patching to use the `pay.dev:8000` API but I'll wait until the layout patch has landed
- after merging the service patch, I'll update the payments-ui docs on hot reloading to link over here

File: payments_service/settings/base.py [hunk 1]
```diff
@@ -239,3 +250,9 @@
 SUBSCRIPTION_FROM_EMAIL = 'no-reply@mozilla.com'
 
 MANAGEMENT_URL = 'http://pay.dev:8000/management.html'
+
+# A request origin value that CORS should be enabled for.
+# If this is None, CORS will not be enabled at all.
+ENABLE_CORS_FOR_ORIGIN = ('http://pay.webpack:8080'
+                          if os.environ.get('SERVICE_USE_WEBPACK') == '1'
```
Comment (muffinresearch) [inline payments_service/settings/base.py]: ~~Shouldn't this be in the settings file and default to being set by the environment var?~~ Ah ignore me this is the settings file. Should of had a coffee before reviewing.

Could it take any boolean?

Comment (muffinresearch) [inline payments_service/settings/base.py]: I wonder if you should also utilise DEBUG as part of the condition tool as an extra safety net?

**Comment by muffinresearch** [comment]
r+wc

**Pull request merged**